storage/replica.go

```diff
@@ -1660,3 +1677,49 @@ func (r *Replica) maybeAddToSplitQueue() {
 		r.store.splitQueue.MaybeAdd(r, r.store.Clock().Now())
 	}
 }
+
+// processReplicaCmd get a raft command from command queue and
+// call processRaftCommand to execute in a loop until there is
+// not any command in the queue for 5 seconds.
```

Comment:
Because I considered that if there are so many ranges, there will be some many goroutines running even if they not working. It would be a waste of system resources and scheduling time.